File: app/src/main/java/reschikov/geekbrains/androidAdvancedLevel/weatherApplication/data/opencage/GeoCoordinates.java
package reschikov.geekbrains.androidadvancedlevel.weatherapplication.data.opencage;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class GeoCoordinates {

    @SerializedName("rate")
    @Expose
    private Rate rate;

    @SerializedName("results")
    @Expose
    private List<Result> results;

    @SerializedName("status")
    @Expose
    private Status status;

    public Rate getRate() {
        return rate;
    }

    public List<Result> getResults() {
        return results;
    }

    public Status getStatus() {
        return status;
    }
}

File: app/src/main/java/reschikov/geekbrains/androidadvancedlevel/weatherapplication/data/network/model/data/opencage/GeoCoordinates.kt
package reschikov.geekbrains.androidadvancedlevel.weatherapplication.data.network.model.data.opencage

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class GeoCoordinates(@SerializedName("rate") @Expose val rate: Rate,
                          @SerializedName("results") @Expose val results: List<Result>,
                          @SerializedName("status") @Expose val status: Status)

